ansible/edusharing-crawler-playbook.yml

```diff
@@ -8,11 +8,15 @@
     kubernetes.core.helm_repository:
       name: dbildungscloud
       repo_url: "https://hpi-schul-cloud.github.io/helm-charts-registry"
+    check_mode: false
```

Comment:
Can you leave a comment about the reason for disabling the check mode and change notification here